components/typography/Base/index.tsx

```diff
@@ -489,26 +489,21 @@ const Base = React.forwardRef<HTMLElement, BlockProps>((props, ref) => {
               expanded={expanded}
               miscDeps={[copied, expanded, copyLoading, enableEdit, enableCopy]}
             >
-              {(node, canEllipsis) => {
-                let renderNode: React.ReactNode = node;
-                if (node.length && canEllipsis && !expanded && topAriaLabel) {
-                  renderNode = (
-                    <span key="show-content" aria-hidden>
-                      {renderNode}
-                    </span>
-                  );
-                }
-
-                const wrappedContext = wrapperDecorations(
+              {(node, canEllipsis) =>
+                wrapperDecorations(
                   props,
                   <>
-                    {renderNode}
+                    {node.length && canEllipsis && !expanded && topAriaLabel ? (
```

Comment:
这里条件渲染建议不要写数字：

https://zh-hans.react.dev/learn/conditional-rendering#logical-and-operator-

```suggestion
                    {node.length > 0 && canEllipsis && !expanded && topAriaLabel ? (
```

Reply:
![image](https://github.com/ant-design/ant-design/assets/49217418/20b5fd85-bff7-4989-8bc1-da938bcecb6d)